fabric-common/types/index.d.ts

```diff
@@ -245,6 +245,15 @@ export class Proposal extends ServiceAction {
 	public compareProposalResponseResults(proposalResponses: any[]): boolean;
 }
 
+export class BlockDecoder {
+	constructor();
+	public static decode(blockBuf: Buffer): fabproto6.common.Block;
+	public static decodeBlock(blockProto: object): fabproto6.common.Block;
```

Comment:
This will take a `fabproto6.common.IBlock` as its argument and version of that completely unpacked to a plain JavaScript object... or (a little confusingly) just return an `fabproto6.common.IBlock` too